app/src/main/java/org/piwigo/ui/main/MainActivity.java

```diff
@@ -165,6 +170,17 @@ private void itemSelected(int itemId) {
                 startActivity(new Intent(getApplicationContext(),
                         ManageAccountsActivity.class));
                 break;
+            case R.id.nav_create_album:
+                AlertDialog.Builder builder = new AlertDialog.Builder(MainActivity.this, R.style.Theme_Piwigo_AlertDialog);
+                builder.setTitle("Create album");
```

Comment:
Also here we should have localization...

Reply:
See the answer above